internal/website/gatherexamples/gatherexamples.go

```diff
@@ -14,22 +14,24 @@
 
 // Command gatherexamples extracts examples in a Go module into a JSON-formatted
 // object. This is used as input for building the Go CDK Hugo website.
-// Examples must include a comment that starts with "// This example is used in"
+//
+// Examples must include a comment
+// "// PRAGMA(gocloud.dev): Package this example for gocloud.dev."
```

Comment:
s/Package/Prepare/?
s/Package/Use/?
since "package" has other meaning in go

Reply:
Yea, good point.

How about this?

```
// PRAGMA(gocloud.dev): This example is used on gocloud.dev; please ignore PRAGMA comments.
```

Reply:
What about this?
PRAGMA(gocloud.dev): This example is adjusted with PRAGMA comments for displaying on gocloud.dev.
or shorter:
PRAGMA(gocloud.dev): This example is adjusted with PRAGMA comments for gocloud.dev.

Reply:
Trying this:

`This example includes PRAGMA comments that adjust how it is shown on gocloud.dev.`

Reply:
Actually that looked weird in examples with no `PRAGMA` statements other than that one.

Trying this:

```
This example is used on gocloud.dev; PRAGMA comments adjust how it is shown and can be ignored.
```

Reply:
This is definitely descriptive and meaningful. No confusion. Does it look too weird in godocs due to the lines being so long?

Reply:
I think it looks OK, at least on my local godoc.